.github/workflows/build.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install -U pip
-          python -m pip install -U docutils
+          python -m pip install -U docutils pygments
```

Comment:
Perhaps we should create `requirements.txt` and pin the versions of`docutils` and `pygments`.

Reply:
Done!

Reply:
Hi @ofek - not sure if youve seen there is an existing reqs file in the repo for the sphinx work? https://github.com/python/peps/blob/master/requirements.txt -- would your one conflict with the pre-existing one?

A

Reply:
Perils of reading commit info from emails and not checking the latest! Feel free to bump the docutils/sphix reqs, they are current as of April 2020, so 0.17.z / 4.y.z may be more appropriate.

A

Reply:
Pushed commit!